public/index.pug

```diff
@@ -29,22 +29,40 @@ html(lang="en-US")
         li(class="header__list-item")
-          a(href="https://unpkg.com/browse/simple-icons/") Unpkg
+          a(class="header__list-icon" title="Unpkg (CDN)" href="https://unpkg.com/browse/simple-icons/" rel="noopener") 
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="M 12.000002,-1.0439e-4 A 12.000002,12.000002 0 0 0 0,11.999898 12.000002,12.000002 0 0 0 12.000002,23.9999 12.000002,12.000002 0 0 0 24.000005,11.999898 12.000002,12.000002 0 0 0 12.000002,-1.0439e-4 Z m 0,2.00178449 A 10.008923,10.008923 0 0 1 22.008925,12.010603 10.008923,10.008923 0 0 1 12.000002,22.019525 10.008923,10.008923 0 0 1 1.9910798,12.010603 10.008923,10.008923 0 0 1 12.000002,2.0016801 Z m 2.140946,3.9286359 -0.03126,4.046388 -0.03329,4.046388 c -0.730063,2.108832 -3.414809,2.108832 -3.992864,0.007 C 10.008601,13.719677 9.9657825,12.23172 9.9657825,9.7482222 V 5.9373383 H 6.8400013 l 0.04314,4.1427307 c 0.053417,4.474577 0.08446,4.742195 0.6615523,5.909011 0.3532561,0.685103 1.2096345,1.445138 1.9696703,1.723461 1.3916151,0.524532 3.6824271,0.524532 5.0740421,0 0.749331,-0.278323 1.552186,-0.975201 1.937556,-1.680642 0.58876,-1.102587 0.654059,-1.627119 0.654059,-6.016059 l 0.0098,-4.0142733 -3.040143,-0.064657 z")
         li(class="header__list-item")
-          a(href="https://github.com/simple-icons/simple-icons/blob/develop/DISCLAIMER.md") Disclaimer
+          a(class="header__list-icon" title="Open Collective" href="https://opencollective.com/simple-icons" rel="noopener") 
```

Comment:
We _could_ remove it from the footer now, what do you think?

Reply:
agreed. will do

Reply:
Initially I thought you meant the disclaimer.
I would keep the Open Collective link in both footer and header. It's not going to hurt to have "calls to action" in a couple places. And it's not obtrusive or anything in the footer either, so... I say let it. At least until we need the space for something else more important.

But since you mentioned the footer.... should we add twitter to the menu as well?

Reply:
Just a suggestion (hence the italics _could_) and I'm fine either way.

> But since you mentioned the footer.... should we add twitter to the menu as well?

I'm a bit on the fence in that regard: all the other links are to _our_ page on the respective platform (excluding the disclaimer), whereas the twitter button in the footer is to tweet about the project which can at least be confusing and may be interpreted as being misleading (I personally wouldn't like it). What do you think?

Reply:
I agree. It would look out of place in the menu.
Let's keep it down there for the time being